.github/workflows/ci-actions.yml

```diff
@@ -203,7 +203,7 @@ jobs:
         run: tar -xzf maven-repo.tgz -C ~
       - name: Build with Maven
         shell: bash
-        run: mvn -B --settings .github/mvn-settings.xml -DskipDocs -Dno-native -Dformat.skip -pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/devtools install
```

Comment:
`-Dno-native` must stem from the early times of Quarkus as I couldn't find any reference to it.
Since I had to introduce a few new `no-native` profiles, I wanted to avoid confusion and so I removed this system property.